code/data_structures/kd_tree/kd_tree/kd_tree_in_js/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "kd-tree-javascript",
+  "version": "1.0.3",
+  "description": "",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ubilabs/kd-tree-javascript.git"
```

Comment:
Looks like you copied the code. This cannot be merged 👍

Reply:
For the JSON object creation it was done! :)
Still I will make furtheer changes for the same! 💯